README.md

```diff
@@ -211,13 +211,13 @@ instead of relying exclusively on the nc-config utility.
 The easiest way to install is through pip:
 
 ```shell
-pip install netcdf4
+pip install netCDF4
 ```
 
 or, if you are a user of the Conda package manager,
 
 ```shell
-conda install -c conda-forge netcdf4
+conda install -c conda-forge netCDF4
```

Comment:
FYI, on conda-forge:
```
❯ conda list netcdf4
# packages in environment at /Users/rmay/miniconda3/envs/py39:
#
# Name                    Version                   Build  Channel
netcdf4                   1.5.6           nompi_py39h353b61e_101    conda-forge
```

Reply:
conda is case insensitive to allow 1-1 correspondence with PyPI and docs. Either way is fine.